packages/flyout/src/Flyout.js

```diff
@@ -269,12 +275,20 @@ export default class Flyout extends Component {
   }
 
   renderPanel({ transitionStatus }) {
-    const { panel } = this.props;
+    const { panel, ...otherProps } = this.props;
+    const { className } = otherProps;
 
     if (typeof panel === "function") {
       return panel({
         ...this.createPanelPayload(),
-        transitionStatus
+        transitionStatus,
+        className: className
+          ? cx(
+              className
+                .split(" ")
+                .map(name => `${name.trim()}-container__panel`)
```

Comment:
Do you mean we will pass what user give to `Flyout`, like "my-class" to `Panel`?  Then the `Panel` will have same class names with `Flyout`... Or you mean we add suffix like `-panel`, `-panel__container` and then append to `PanelPresenter` in PanelContainerPresenter.js/PanelPresenter.js ? 

The big concern is: if we think `Panel` as a normal component, it should append `className` from its own props to its outer wrapper. (of course it can add classnames with suffix to its children DIVs). That means, in `Flyout`, if we just pass  `Flyout.props.className` to `Panel`, the `Panel` will have same class names with `Flyout`, which will break styles 😿